hyperpackage/hyperpackage/flavor/automl/unpack_hp_univariate_ts.py

```diff
@@ -0,0 +1,49 @@
+"""This script unpacks hyperpack onnx model artifacts that were 
+originated from Hypergiant's AutoML Univariate TimeSeries.
+NOTE: Since time series is comprised of multiple model artifacts,
+this script currently only opens up 1 model artifact at a time. 
+Therefore, if multiple model artifacts need to be unpacked, a for-loop
+or iteration over multiple artifact files is required using the code herein.
+"""
+
+from icecream import ic
+import onnx
+from onnx2torch import convert
+
+from utilities import UnpackHyperpack
```

Comment:
When trying to import the `UnpackAutoMLHyperpack` class, I think this is the place where the "import error" is occurring. Might fix it to something like this:

```suggestion
from hyperpackage.utilities import UnpackHyperpack
```